samples/net/sockets/big_http_download/sample.yaml

```diff
@@ -8,3 +8,12 @@ tests:
     min_ram: 32
     min_flash: 128
     tags: net socket
+  sample.net.sockets.big_http_download.posix:
+    filter: TOOLCHAIN_HAS_NEWLIB == 1
```

Comment:
Please move the common values between `sample.net.sockets.big_http_download` and `sample.net.sockets.big_http_download.posix` to common block.
So something like this:
```
common:
  harness: net
  tags: net socket
  min_ram: 32
  min_flash: 128
```

Reply:
> Please move the common values between 

Fixed.